### DISCUSSION TITLE
i have a question, is there a lts version of k3s?

### DISCUSSION THREAD

**Discussion by wang-xiaowu** [k3s-io/k3s#7565] [Q&A] [answered]
i have downloaded the version : 1.24.4+k3s1 before
but now i got this
![image](https://github.com/k3s-io/k3s/assets/44340137/61aeae09-18c4-4c24-8467-98149ec4f295)

**Comment by wang-xiaowu** [comment]
sorry, i need repeat my question
i think it only has problem in "rancher.cn"

**Reply by brandond** [reply, marked as answer]
Yes, that is run by a different team that does not monitor this repository. I would ask on the rancher.cn forums.
